includes/class-wc-bfx-pay-gateway.php

```diff
@@ -554,33 +594,13 @@ public function remove_order_details()
     public function email_template($order, $sent_to_admin, $plain_text, $email)
     {
         $order->read_meta_data(true);
-        $metadata = $order->get_meta_data();
-        $invoiceId = null;
-        foreach ($metadata as &$entry) {
-            $data = $entry->get_data();
+        $invoiceId = get_post_meta($order->id, 'bitfinex_invoice_id', true);
```

Comment:
hmm is this gonna be supported in long term? seems more straightforward than $order->get_meta_data() but not sure if it will be long term compatible

Reply:
`get_post_meta` is a standard function for wordpress. In wordpress, each item has a uniq id, and this item can be either post, page, order and so on. So that we can use this function in a long term IMO.

Reply:
ok then let's use `get_post_meta`